server/server.js

```diff
@@ -16,10 +16,12 @@ app.use(bodyParser.urlencoded({ extended: true }));
 // get driver connection
 const dbo = require("./db/conn");
 
-https.createServer({
-  key: fs.readFileSync('privatekey.pem'),
-  cert: fs.readFileSync('fullchain.pem'),
-}, app).listen(port, () => {
+// https.createServer({
```

Comment:
@HaiderZaidiDev I have commented these lines - I am not sure do we need this?  as its giving me error for these two files not found
 - `privatekey.pem`
 - `fullchain.pem`

Reply:
I'm not sure what these do.  Are they related to the MongoDB API perhaps?

Reply:
Hi @devarsh19, we do need these, as they're required for the API server to run in SSL when in production. It's okay to have it out in development but it's needed for the prod branch.

Reply:
oh okay, Thanks for the information @HaiderZaidiDev